R/derive_param_confirmed_resp.R

```diff
@@ -312,11 +312,18 @@ derive_param_confirmed_resp <- function(dataset,
     )
   }
 
+  # Check for CR followed by PR (this should not occur in clean data)
+  signal_crpr(
+    source_data,
+    order = vars(ADT)
+  )
+
```

Comment:
See above

Reply:
I am making filter_pd optional in all calling functions, so if CR > PD > PR is observed the checking code would give an error.  So if CR > PD > PR is possible, the code needs to be updated (I think).

Reply:
I wonder how we should update the code. Should we remove all observations after PD and then check the remaining observations?

Reply:
think we need to focus a bit here - admiral is not a data cleaning service and we could create endless work for ourselves. we should just provide assertions that either may help spot user input errors or something that might impact the feasibility of an analysis derivation. we are only checking for CR->PR here because of the impact it has to the confirmed BOR/response analysis. if a user chooses to derive confirmed BOR without filtering out PDs then in my mind it's at their own risk that there may be such data issues that need to be considered. in reality what they're deriving there isn't even RECIST confirmed BOR anyway, as you should always stop at PD.

Reply:
in a nutshell i'm fine if the code errors for CR->PD->PR if thats the way its been set-up but i'd equally be OK if it didn't. whatever's the least amount of extra effort for you

Reply:
At the moment it issues a warning if CR -> PD -> PR occurs.

Reply:
that works for me, technically its a data issue too, but just one i'm personally less concerned of